backup-stores/s3/src/test/java/io/camunda/zeebe/backup/s3/LocalstackBackupStoreIT.java

```diff
@@ -20,21 +21,40 @@
 
 @Testcontainers
 final class LocalstackBackupStoreIT implements S3BackupStoreTests {
+  private static final String BUCKET_NAME = RandomStringUtils.randomAlphabetic(10).toLowerCase();
 
   @Container
   private static final LocalStackContainer S3 =
       new LocalStackContainer(DockerImageName.parse("localstack/localstack"))
           .withServices(Service.S3);
 
-  private static S3AsyncClient client;
+  private S3AsyncClient client;
   private S3BackupStore store;
   private S3BackupConfig config;
 
+  @BeforeAll
+  static void setupBucket() {
+    final var config =
+        new Builder()
+            .withBucketName(BUCKET_NAME)
+            .withEndpoint(S3.getEndpointOverride(Service.S3).toString())
+            .withRegion(S3.getRegion())
+            .withCredentials(S3.getAccessKey(), S3.getSecretKey())
+            .withApiCallTimeout(null)
+            .forcePathStyleAccess(false)
+            .withCompressionAlgorithm(null)
+            .build();
+    try (final var client = S3BackupStore.buildClient(config)) {
+      client.createBucket(CreateBucketRequest.builder().bucket(config.bucketName()).build()).join();
+    }
+  }
+
   @BeforeEach
-  void setupBucket() {
+  void setup() {
     config =
         new Builder()
-            .withBucketName(RandomStringUtils.randomAlphabetic(10).toLowerCase())
+            .withBucketName(BUCKET_NAME)
+            .withBasePath(RandomStringUtils.randomAlphabetic(10).toLowerCase())
```

Comment:
:question: Do we have a test with no basePath?

Reply:
The integration tests are not using it yet but I'll add an explicit test :+1: